README.md

```diff
@@ -8,7 +8,8 @@
 
 |    | Sample Name        | Description                                                                                                                | C#    | TypeScript   
 |:--:|:-------------------|:---------------------------------------------------------------------------------------------------------------------------|:--------|:-------------|
-|1|Hello World            | Microsoft Teams hello world sample app.                                          |[View][app-hello-world#cs]     |[View][app-hello-world#ts]
+|1|Hello World            | Microsoft Teams hello world sample app.                                           |[View][app-hello-world#cs]     |[View][app-hello-world#ts]
+|1|Teams Virtual Assistant| Customized virtual assistant template to support teams capabilities.              |[View][msteams-virtual-assistant#cs]|
```

Comment:
Could you please move this to [Additional Sample](https://github.com/OfficeDev/Microsoft-Teams-Samples#additional-samples)? Also please correct the hyperlink.

Reply:
updated

Reply:
Could you please update following? 
Sequence of sample - In table auto numbering doesn't work.
Hyperlink - I still see [View][msteams-virtual-assistant#cs] in Read me file.